dojo/importers/reimporter/reimporter.py

```diff
@@ -69,6 +70,8 @@ def process_parsed_findings(
             deduplication_algorithm,
         )
         for item in items:
+            if apply_tags_to_findings:
+                item.tags.set(tags)
```

Comment:
Is this needed here as well as the `reimport_scan` function? I think it would be best to remove this instance of adding the tags the findings

Reply:
I do think it is a good feature to have on re-imports.  It provides users the opportunity to apply tags that either were not available or they missed during the initial import.